examples/src/Viewer.tsx

```diff
@@ -1029,6 +1043,9 @@ class Viewer extends React.Component<InputParams, ViewerState> {
                                 ? this.onRecordedMovie
                                 : undefined
                         }
+                        onFollowObjectChanged={this.handleFollowObjectData.bind(
```

Comment:
just wondering again if this should be at the SimulariumViewer or Controller level...

Reply:
IMO, viewer level if only because it fits the existing pattern for how the front end subscribes to viewer data.

I could imagine a shift to making the controller a publisher and subscription is done by calling controller methods rather than handing down props, but my intuition is that app shouldn't have a mixed approach to handling this. Does that make sense?